src/main/scala/com.snowplowanalytics.snowplow.analytics.scalasdk/SnowplowEvent.scala

```diff
@@ -14,8 +15,19 @@ object SnowplowEvent {
   case class UnstructEvent(data: Option[SelfDescribingData[Json]]) extends AnyVal
   case class Contexts(data: List[SelfDescribingData[Json]]) extends AnyVal
 
-  implicit final val unstructCirceDecoder: io.circe.Decoder[Contexts] = ???
-  implicit final val unstructCirceEncoder: io.circe.Encoder[Contexts] = ???
-  implicit final val contextsCirceDecoder: io.circe.Decoder[UnstructEvent] = ???
-  implicit final val contextsCirceEncoder: io.circe.Encoder[UnstructEvent] = ???
+  implicit final val schemaVerFullCirceEncoder: io.circe.Encoder[SchemaVer.Full] = deriveEncoder[SchemaVer.Full]
```

Comment:
Semiautomatic decoder will derive decoder for `SchemaVer` as for:

```json
{
  "model": 1,
  "revision": 0,
  "addition": 0
}
```

Which is not correct. `iglu-core-circe` (you can use latest 0.4.0 RCs) includes proper decoders for `SchemaVer`, `SchemaKey` etc